src/pages/my-buses/my-buses.html

```diff
@@ -24,7 +24,7 @@
       <span role="heading">Your Routes</span>
       <button ion-button outline item-right aria-label="Add favorite routes. Opens a popup that won't get keyboard focus, please navigate to the bottom of the page after pressing" (click)="showRouteModal()">Add Routes</button>
     </ion-item-divider>
-      <ion-item *ngIf="!routes">
+      <ion-item *ngIf="!routes || routes.length == 0">
```

Comment:
I saw this in other files. I could potentially replace those with `isEmpty`, too.